src/02-internuclear-potentials.md

```diff
@@ -0,0 +1,265 @@
+<a href="https://colab.research.google.com/github/project-ida/nuclear-reactions/blob/master/02-internuclear-potentials.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://nbviewer.jupyter.org/github/project-ida/nuclear-reactions/blob/master/02-internuclear-potentials.ipynb" target="_parent"><img src="https://nbviewer.jupyter.org/static/img/nav_logo.svg" alt="Open In nbviewer" width="100"/></a>
+
+
+# Internuclear potentials
+
+
+This notebook looks at different sections of the internuclear potential between two nuclei, specifically two deuterium nuclei as in the case of a molecule of deuterons D2. 
+
+The three potentials considered are:
+1. The numerical Kolos & Wolniewicz potential (and the parametrized Morse potential)
+2. The parametrized Coulomb potential
+3. The parametrized Woods-Saxon nuclear potential
+
+
+## Loading libraries and helper functions
+
+```python
+import pandas as pd
+import numpy as np
+import plotly.express as px
+import plotly.graph_objects as go
+from scipy.constants import pi, epsilon_0, e
+from matplotlib import pyplot as plt
+```
+
+```python
+def bohr_to_pm(length):
+    return length*5.292e-11*1e12
+
+def pm_to_bohr(length):
+    return length/(5.292e-11*1e12)
+
+def hartree_to_ev(energy):
+    return energy*27.211
+```
+
+## Kolos & Wolniewicz (KW): numerically determined interatomic potential
+
+From Koonin & Nauenberg (KN) 1989: _"we have taken for the diatomic molecular potential V(r) the best available numerical calculation in the Born-Oppenheimer approximation, due to Kolos and Wolniewicz"_
+
+The KW reference is (the numerically determined potential is given in Table III): 
+> Kolos, W., & Wolniewicz, L. (1964). Accurate adiabatic treatment of the ground state of the hydrogen molecule. The Journal of Chemical Physics, 41, 3663–3673.
+
+And KN 1989 describe further: _"For 1.1 < r < 3 this potential is well approximated by the Morse potential"_ (Note that 1.1 bohr = 58 pm and 3 bohr = 159 pm)
+
+The Morse potential is a simple parameterized potential. It is usually given as: 
+
+$$V(r)=D_{e}(e^{-2a(r-r_{e})}-2e^{-a(r-r_{e})})$$
+
+```python
+kw = pd.read_csv("./data/k-w_potential.csv")
+```
+
+```python
+def morsepot_au(r): # in atomic units
+    a = 0.4
+    return (1+0.1745)*(np.exp(-2*a*(r-1.4))-2*np.exp(-a*(r-1.4))) 
```

Comment:
From reading the Koonin paper, it looks like the entire potential needs to be shifted down by -1, i.e.
```
(0.1745)*(np.exp(-2*a*(r-1.4))-2*np.exp(-a*(r-1.4))) - 1
```
If I use a=1.04 as in the above comment I get a much better fit to the KW potential energy